content/news/playtest-20150801.md

```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
```

Comment:
No, HTML compresses multiple whitespace characters into a single space (even newlines etc.), so it doesn't matter for rendering at all. It just better for reading the source (look up [English punctuation](https://en.wikipedia.org/wiki/Space_(punctuation)#Spaces_between_sentences)).